### PR TITLE
[RSDK-9086] Move from git lfs to artifact

### PR DESCRIPTION
**Pull request by seanavery**
## Description

Git LFS costs money for bandwidth! We have an in-house artifact system for large files. This PR moves tests over to goutils artifact.

### CODE REVIEW

**Comment by hexbabe** [comment]
This is cool! I'll review soon

**Review by randhid — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -45,6 +45,7 @@ endif
 CGO_CFLAGS := -I$(FFMPEG_BUILD)/include
 GOFLAGS := -buildvcs=false
 export PKG_CONFIG_PATH=$(FFMPEG_BUILD)/lib/pkgconfig
+export PATH := $(PATH):$(shell go env GOPATH)/bin
```
Comment (hexbabe) [inline Makefile:48]: What are we using `PATH` for here?

File: tests/config_test.go [hunk 2]
```diff
@@ -57,13 +58,13 @@ func cleanVideoStoreDir() error {
 	if err != nil {
 		return err
 	}
-	videoStoreDir := filepath.Join(currentDir, "video-storage")
+	videoStoreDir := filepath.Join(currentDir, "../.artifact/data")
```
Comment (hexbabe) [inline tests/config_test.go:61]: Should we use `artifactStoragePath`

**Review by hexbabe — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (seanavery) [reply Makefile:48]: Just so that bins in go path are available (for `artifact` calls).

File: tests/config_test.go (hunk 2 above)
Comment (seanavery) [reply tests/config_test.go]: Ty. Changed.

**Review by hexbabe — APPROVED** [review]
Thanks!!

**Pull request merged**